benchmarks/build.gradle

```diff
@@ -1,14 +1,14 @@
-apply plugin: "com.github.johnrengelman.shadow"
 apply plugin: "me.champeau.gradle.jmh"
 
 
 dependencies{
-  compile project(':reactor-core')
-
-  compile "org.openjdk.jmh:jmh-core:1.21"
-  compile "org.openjdk.jmh:jmh-generator-annprocess:1.21"
-  compile "io.projectreactor.addons:reactor-extra:3.3.0.BUILD-SNAPSHOT"
+  jmh project(':reactor-core')
 
+  jmh "org.openjdk.jmh:jmh-core:1.21"
+  jmhAnnotationProcessor "org.openjdk.jmh:jmh-generator-annprocess:1.21"
+  jmh "io.projectreactor.addons:reactor-extra:3.3.0.BUILD-SNAPSHOT", {
+    exclude group: 'io.projectreactor', module: 'reactor-core'
```

Comment:
for some reason, IntelliJ IDEA was using the binary dependency instead of a project one o_O